runtime/runtime_impl_jetty12/src/main/java/com/google/apphosting/runtime/jetty/JettyServletEngineAdapter.java

```diff
@@ -104,12 +106,21 @@ public void start(String serverInfo, ServletEngineAdapter.Config runtimeOptions)
       threadPool.setVirtualThreadsExecutor(VirtualThreads.getDefaultVirtualThreadsExecutor());
       logger.atInfo().log("Configuring Appengine web server virtual threads.");
     }
+
+    // The server.getDefaultStyleSheet() returns is returning null because of some classloading issue,
```

Comment:
Can you be either more or less vague than "some classloading issue"

Reply:
Turned this into a TODO to investigate why we need to do this, otherwise I could just remove the comment all together.